meson.build

```diff
@@ -95,9 +95,10 @@ endif
 
 prefixdir = get_option('prefix')
 bindir = join_paths(prefixdir, get_option('bindir'))
+libdir = join_paths(prefixdir, get_option('libdir'))
 datadir = join_paths(prefixdir, get_option('datadir'))
 docdir = join_paths(datadir, 'doc/casync')
-protocoldir = join_paths(prefixdir, 'lib/casync/protocols')
+protocoldir = join_paths(libdir, 'casync/protocols')
```

Comment:
libdir is the wrong place for executable binaries. If anything it should be libexecdir, but not libdir here.